.cypress/integration/app_analytics_test/app_analytics.spec.js

```diff
@@ -479,8 +483,7 @@ describe('Separate from other plugins', () => {
     cy.visit(
       `${Cypress.env('opensearchDashboards')}/app/observability-dashboards#/`
     );
-    cy.get('[data-test-subj="operationalPanelsActionsButton"]', { timeout: timeoutDelay }).click();
-    cy.get('[data-test-subj="addSampleContextMenuItem"]', { timeout: timeoutDelay }).click();
+    cy.get('.euiButtonContent').contains('Add samples').click();
```

Comment:
Does this element have a `data-test-subj`?

Reply:
It does not